packages/genesys-spark-chart-components/src/components/beta/gux-visualization/tests/d3-rs-intl.spec.ts

```diff
@@ -0,0 +1,2364 @@
+import { time } from '@redsift/d3-rs-intl';
+
+describe('@redsift/d3-rs-intl', () => {
+  describe('time snapshot tests', () => {
```

Comment:
This test is pretty pointless but it gives us something to reference if anyone comes to us with bugs in the localisations